lib/registries/addon/drafts/draft/template.hbs

```diff
@@ -21,9 +21,50 @@
             <layout.heading local-class='Heading'>
                 <div local-class='Title'>{{t 'registries.drafts.draft.form.new_registration'}}</div>
             </layout.heading>
-            <layout.left local-class='Left'>
-                {{!-- Sidenav: page headings, validations --}}
-            </layout.left>
+            
+            <layout.leftNav as |leftNav|>
+                {{#each draftManager.pageManagers as |pageManager index|}}
+                    <PageLink
+                        @pageManager={{pageManager}}
+                        @pageIndex={{index}}
+                        @currentPage={{draftManager.currentPage}}
+                        @draftId={{draftManager.draftId}}
+                        as |pageLink|
+                    >
+                        <leftNav.link
+                            data-analytics-name='{{pageLink.IndexWithSlug}}'
+                            class={{local-class pageLink.validationClass}}
+                            @route='registries.drafts.draft'
+                            @models={{pageLink.models}}
+                            @icon={{pageLink.icon}}
+                            @label={{pageManager.pageHeadingText}}
+                            @isCurrentPage={{pageLink.pageIsActive}}
+                            as |link|
+                        >
+                            <link.icon />
+                            <link.label
+                                local-class={{if (eq pageLink.validationClass 'Unvisited') 'UnvisitedDark'}}
```

Comment:
Instead of doing this condition, it'd be better to statically add a local class here and then apply the dark color using nesting:
```hbs
<link.label local-class='Label' />
```
```scss
.Unvisited {
    .Label {
        color: $dark-slate-gray;
    }
}
```